app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    operator, existing_file, new_file = command.split(" ")
+    if operator == "cp" and new_file != existing_file:
+        with (open(existing_file, "r") as file_in,
+              open(new_file, "w") as file_out):
+            for line in file_in:
+                file_out.write(line + "\n")
```

Comment:
The code currently adds an extra newline character when writing each line to the new file. The `line` variable already contains a newline character at the end, so adding another '\n' will result in double newlines in the copied file. Remove the '+ "\n"' to fix this issue.